### DISCUSSION TITLE
api not working

### DISCUSSION THREAD

**Discussion by ameerfayiz** [serverpod/serverpod#2469] [Q&A] [answered]
### Discussed in https://github.com/serverpod/serverpod/discussions/2402

<div type='discussions-op-text'>

<sup>Originally posted by **ameerfayiz** June 20, 2024</sup>
why is this post request not working now? it used to work in postman in earlier versions of serverpod. but now, its returning unauthorized error in 2.0.0 version. what's the format of  "auth" that i should give in 2.0.0 onwards ??

```
{   
   "method" : "updateAirportList",
   "auth": "{{keyId}}:{{key}}",
   "Airports": [..]
}
```

@vlidholt @SandPod @Isakdl @TJ-coding @Aygistov @letscodetpl @gstuntz @NeroSong @CordMemescape

Basically this is my intention:
I have to write a script to update list of rows to the database . Earlier, i wrote the script in python using above request format and it was working. But now i am getting 401 error . How can i do the same it in python or dart? Please reply ASAP. Urgent 🦺</div>

**Comment by SandPod** [comment, marked as answer]
Hey @ameerfayiz!

The format for the authentication key has not been deliberately changed.

However, a breaking change in 2.0.0 is that the authentication module now also contains the authentication handler that needs to be set on your Serverpod instance.

Did you perform this step in the migration guide: https://docs.serverpod.dev/upgrading/upgrade-to-two#changes-to-authentication?
